bin/sk-tests

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-karma start --single-run "$@"
+sk-lint && karma start --single-run "$@"
```

Comment:
Yeah, but ideally you won't have any errors ;). It happens prior to commit anyway, so I might just leave it here. It doesn't do it in watch mode and this will ensure we catch stuff at build time, too.